docs/getting-started/install/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Install Shopify App CLI
+section: getting-started
+toc: false
+redirect_from: "/install/"
+---
+
+Shopify App CLI can be installed using a variety of package managers.
+
+### Ruby gem
+
+```console
+$ gem install shopify-cli
+```
+
+### apt (Debian, Ubuntu)
+
+You’ll need to install a downloaded .deb file with an explicit version number. Make sure you check the [releases page](https://github.com/Shopify/shopify-app-cli/releases) to make sure you install the latest package.
+
+```console
+$ apt install shopify-cli-x.y.z.deb
+```
+
+### Brew (MacOS)
+
+You’ll need to run `brew tap` first to add Shopify’s third-party repositories to Homebrew.
+
+```console
+$ brew tap shopify/shopify && brew install shopify-cli
```

Comment:
I suggest we break these into 2 separate commands, as the user only needs to tap once to be able to install / update / uninstall the formula.